doc/source/reference/c-api/types-and-structures.rst

```diff
@@ -744,6 +745,16 @@ PyArrayMethod_Context and PyArrayMethod_Spec
       An array of descriptors for the ufunc loop, filled in by
       ``resolve_descriptors``. The length of the array is ``nin`` + ``nout``.
 
+   .. c:member:: void *parameters
+
+      A pointer to a structure containing any parameters needed by the
```

Comment:
Maybe insert "runtime" in front of "parameters"?

Reply:
Done.